packages/widget.client-react/src/context.ts

```diff
@@ -25,6 +26,10 @@ import {
 } from "@osdk/widget.client";
 import React, { useContext } from "react";
 
+export type ExtractObjectSet<OT extends ObjectType> = NonNullable<
+  NonNullable<OT["__DefinitionMetadata"]>["objectSet"]
```

Comment:
Is there anywhere that will complain if you pass in an object type that happens to not actually have `__DefinitionMetadata.objectSet` defined for example if TS OSDK made a break in the generation unintentionally?